app/services/courses_query_logger.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class CoursesQueryLogger
+  def initialize(applied_filters, scope, explain: false)
+    @applied_filters = applied_filters
+    @scope = scope
+    @log_tag = self.class.name
+    @logger = Rails.logger
```

Comment:
These 2 variables look like they don't need to be _variables_.

```ruby
def log_tag
  self.class.name
end

def logger
  Rails.logger
end
```

Reply:
Done